x-pack/plugins/ml/public/application/components/data_grid/common.ts

```diff
@@ -110,7 +111,10 @@ export const getDataGridSchemasFromFieldTypes = (fieldTypes: FieldTypes, results
       schema = 'numeric';
     }
 
-    if (field.includes(`${resultsField}.${FEATURE_IMPORTANCE}`)) {
+    if (
```

Comment:
Is it possible to remove the `ml.top_classes` and `ml.feature_importance` leaf node fields from the data grid column picker modal e.g. `ml.top_classes.class_name`? Ideally we shouldn't let the user select the leaf nodes for display in the table as those columns will be blank:

![image](https://user-images.githubusercontent.com/7405507/83867502-6ba5aa80-a721-11ea-8a7b-3bd335271649.png)